ipaserver/plugins/ca.py

```diff
@@ -208,6 +208,9 @@ def set_certificate_attrs(entry, options, want_cert=True):
                         ca=entry['cn'][0])
                 else:
                     raise e
+            except errors.NotFound:  # pki v2 api
+                msg = messages.LightweightCACertificateNotAvailable(
+                    ca=entry['cn'][0])
```

Comment:
I guess this is from: ipa ca-show foo, where foo doesn't exist? I can't reproduce an unhandled exception.

Reply:
`foo` exists, but when key has not been replicated.  It did not result in *unhandled* exception, but the desired behaviour is still to see the CA properties and a message.  Without this catch, you would only see the message.